web/router.ex

```diff
@@ -26,14 +26,20 @@ defmodule Designator.Router do
     put "/users/:id/add_seen_subject", UserController, :add_seen_subject
   end
 
+  # Ignore Phoenix routing errors
+  defp handle_errors(_conn, %{reason: %Phoenix.Router.NoRouteError{}}), do: :ok
+
+  # report other request errors to rollbar
   defp handle_errors(conn, %{kind: kind, reason: reason, stack: stacktrace}) do
     Rollbax.report(kind, reason, stacktrace)
-    case Poison.encode(%{kind: kind, reason: reason}) do
```

Comment:
Do we not need this part of code anymore?  with `send_resp`?

Reply:
Nope as we aren't manually sending a response anymore - now we are letting the system respond with "Server internal error" and json payload. 

Note that later in the comments I've switched to use the json responder https://hexdocs.pm/phoenix/1.2.5/Phoenix.Controller.html#json/2 vs the send_resp in the comments below. 

The `json` call encodes the data before sending the payload.

Reply:
Ah got it got it 👍